v2/dtos/common/version.go

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package common
+
+// VersionRequest defines the request associated with the /version endpoint.
+// This object and its properties correspond to the VersionRequest object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/VersionRequest
+type VersionRequest struct {
+	RequestID string `json:"requestId" validate:"required,uuid"`
+}
+
+// VersionResponse defines the latest version supported by the service.
+// This object and its properties correspond to the VersionResponse object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/VersionResponse
+type VersionResponse struct {
+	BaseResponse `json:",inline"`
+	Version      string `json:"version"`
+}
+
+// VersionSdkResponse defines the latest sdk version supported by the service.
+// This object and its properties correspond to the VersionSdkResponse object in the APIv2 specification:
+type VersionSdkResponse struct {
+	BaseResponse `json:",inline"`
```

Comment:
This needs to be `VersionResponse`, (instead of `BaseResponse`) since the App Services and Device Services (via SDKs) return both `Version` and `SdkVersion`